backend/tracim_backend/config.py

```diff
@@ -758,12 +855,33 @@ def _set_default_app(self, enabled_app_list: typing.List[str]):
         # app_list is updated.
         update_validators()
 
+    # INFO - G.M - 2019-04-05 - Others methods
+    def _check_consistency(self):
```

Comment:
@buxx Can you check this method, its one of the important point. I do not do a specific error because Attribute Error return seems to me clear, but i want feedback about it

Reply:
I think you can add `try except` where `_check_consistency` is used to be able to print an obvious message for user.

Reply:
done.